### ISSUE TITLE
Parallel processing

### ISSUE THREAD

**Issue by ivank** [chrisbenincasa/berglas-node#6]
Hi @chrisbenincasa Thanks for merging the #5 PR but I've just noticed it hasn't been released to NPM yet. I can use GitHub directly of course like this, but its not very pretty I'm afraid.

```
"berglas-node": "https://github.com/chrisbenincasa/berglas-node.git#a855403612d71052f8f1462665cde4640a729a99",
```

Is there anything preventing master to be published on NPM? Anything I can help with?

**Comment by chrisbenincasa** [comment]
My apologies! Must've slipped my mind. The new version should be published at `1.0.4` now. Sorry again and let me know if you have any issues!

**Comment by ivank** [comment]
Yeah it looks like everything's peachy, massive thanks for the great package!
